modules/osd-intro.adoc

```diff
@@ -42,17 +42,8 @@ Other enhancements to Kubernetes in {product-title} include improvements in soft
 [id="telemetry_{context}"]
 == Internet and Telemetry access for {product-title}
 
-In {product-title}, you require access to the Internet to install your cluster. The Telemetry service, which runs by default to provide metrics about cluster health and the success of updates, also requires Internet access. If your cluster is connected to the Internet, Telemetry runs automatically, and your cluster is registered to the {console-redhat-com}.
+In {product-title}, you require access to the internet to install and upgrade your cluster.
 
-Once you confirm that your Red Hat OpenShift Cluster Manager inventory is correct, either maintained automatically by Telemetry or manually using OCM, use link:https://access.redhat.com/documentation/en-us/subscription_central/2020-04/html/getting_started_with_subscription_watch/con-how-to-select-datacollection-tool_assembly-requirements-and-your-responsibilities-ctxt#red_hat_openshift[subscription watch] to track your {product-title} subscriptions at the account or multi-cluster level.
+Telemetry is the term that Red Hat uses to describe the information being sent to Red Hat by the OpenShift Telemeter Client. Lightweight attributes are sent from {product-title} clusters to Red Hat to enable subscription management automation, monitor the health of clusters, assist with support, and improve customer experience.
 
-You must have Internet access to:
-
-* Access the {console-redhat-com} page to download the installation program and perform subscription management. If the cluster has Internet access and you do not disable Telemetry, that service automatically entitles your cluster.
-
-* Obtain the packages that are required to perform cluster updates.
-
-[IMPORTANT]
-====
-If your cluster cannot have direct Internet access, you can perform a restricted network installation on some types of infrastructure that you provision. During that process, you download the content that is required and use it to populate a mirror registry with the packages that you need to install a cluster and generate the installation program. With some installation types, the environment that you install your cluster in will not require Internet access. Before you update the cluster, you update the content of the mirror registry.
-====
+The Telemetry service runs automatically and your cluster is registered to the {OCM}. In {product-title}, remote health reporting is always enabled and you cannot opt out.
```

Comment:
remote health reporting is enabled by default and unlike with {openshift container platform} you cannot opt out, as .... << why... does SRE use this? elaborate.

Reply:
@sferich888 thank you for your review and for highlighting this.

@0kashi or @wgordon17, can you please provide some additional detail about why our SRE team requires this? What else would you like to see in this section about that? Thanks!

Reply:
@sferich888 the reporting lands in telemeter, which gets aggregated and fed into OCM as our primary management platform. Without the reporting, we have no insights into the cluster from OCM, and can't effectively support it.

Reply:
@wgordon17 thank you for describing this. I have updated this paragraph to include a sentence on how the remote health information is required to enable the SRE team to effectively support OSD clusters.